### PR TITLE
fix(capture-sdk):

### PR DESCRIPTION
**Pull request by AdnanYupi**
- Fix add pages leads to OnBoarding page

PIA-3442

[PIA-3442]: https://ginis.atlassian.net/browse/PIA-3442?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java [hunk 1]
```diff
@@ -470,7 +470,7 @@ private void showFragment() {
     }
 
     private void showOnboardingIfRequested() {
-        if (shouldShowOnboarding()) {
+        if (shouldShowOnboarding() && !mAddPages) {
             startOnboardingActivity();
```
Comment (zladzeyka) [inline capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java:474]: [Clarification] what's the purpose of mAddPages?

**Review by zladzeyka — APPROVED** [review]
Looks good!

File: capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java (hunk 1 above)
Comment (AdnanYupi) [reply capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraActivity.java:474]: @zladzeyka mAddPages is a variable that tells us if Camera Screen is launched from MultiPageReview Screen

**Review by a-szotyori — APPROVED** [review]
Thanks!

**Pull request merged**